package-lock.json

```diff
@@ -8402,7 +8423,6 @@
           "version": "1.1.11",
           "resolved": "",
           "integrity": "sha512-iCuPHDFgrHX7H2vEI/5xpz07zSHB00TpugqhmYtVmMO6518mCuRMoOYFldEBl0g187ufozdaHgWKcYFb61qGiA==",
-          "optional": true,
```

Comment:
Few `"optional": true,`'s removed, can you double check youre on the right npm please :) if you are, ignore this :)